03_Basic SQL/Upgrate 1.0 - 1.1.sql

```diff
@@ -0,0 +1,13 @@
+IF OBJECT_ID(N'[Northwind].[CreditCards]') IS NULL
```

Comment:
I think you wanted to call these files "Update" :)

Reply:
yes, thank you)